new-dti-website/src/app/courses/page.tsx

```diff
@@ -0,0 +1,294 @@
+                  <span className="text-[#877B7B]">Teaching the</span>{' '}
+                  <span className="italic">community</span>
+                </h2>
+                <p className="text-md md:text-xl font-thin text-slate-300">
+                  A project team is meant, above all, to be a learning experience. Given our mission
+                  of{' '}
+                  <span className="font-black text-lg md:text-2xl text-white">
+                    community impact
+                  </span>
+                  , we want to help everyone{' '}
+                  <b className="font-black text-lg md:text-xl text-white">learn and grow</b> through
+                  our training course in{' '}
+                  <b className="font-black text-md md:text-xl text-white">product development</b>.{' '}
+                </p>
+              </div>
+            </div>
+            <div className="relative top-[-250px]">
+              <RedBlob className={'right-[-250px]'} intensity={0.3} />
+            </div>
+          </div>
+        </section>
+
+        {/* WRAPPER */}
+        <div id="Wrapper" className="flex flex-col gap-y-28 bg-[#EDEDED] text-black ">
+          {/* LOGO SECTION */}
+          <section id="Trends and Web Development">
+            <div className="min-h-[60vh] flex flex-col pl-10 pt-20 lg:flex-row lg:items-center lg:justify-around">
+              <div ref={trendsLogoRef} className="sticker">
+                <Image
+                  src={'/icons/courses/trends_logo.png'}
+                  width={450}
+                  height={450}
+                  alt="Trends Logo"
+                  unoptimized
+                  className="w-72 md:w-96 lg:w-[450px]"
+                />
+              </div>
+
+              <div className="flex flex-col lg:w-1/2">
+                <div className="font-black text-sm md:text-xl tracking-wider">
+                  MODERN INDUSTRY-LEADING TECHNOLOGY
+                </div>
+
+                <div className="font-black text-4xl tracking-wider mt-4 md:text-[45px]">
+                  Trends in Web Development
+                </div>
+
+                <div className="text-md md:text-2xl mt-8">
+                  Trends in Web Development in a 1-credit S/U course that showcase modern full-stack
+                  development and best practices used within industry. We cover technologies like
+                  TypeScript, React, Node.js, Firebase, Express and more, all of which are deployed
+                  at scale by leading tech companies
+                </div>
+
+                <div className="flex flex-row gap-x-6 mt-6">
+                  <button
+                    className="rounded-2xl py-2 px-4 md:py-4 md:px-8 bg-[#A52424] text-white text-lg md:text-xl
+          font-black hover:bg-white hover:text-[#A52424] border-4 hover:border-[#A52424] w-fit"
+                  >
+                    Apply Now
+                  </button>
```

Comment:
I talked with Vannessa about the buttons on the page last work sesh. She said that all buttons should, on hover, turn into a darker shade of red (like the ones on the home page). I'm not sure if it is supposed to be different on this page, but we should clarify this next work sesh.

I'm not sure where this one is supposed to link to (I'm guessing the apply page), but the next button should probably link to https://webdev.cornelldti.org/. We can also confirm this with the designers.

Reply:
> I talked with Vannessa about the buttons on the page last work sesh. She said that all buttons should, on hover, turn into a darker shade of red (like the ones on the home page). I'm not sure if it is supposed to be different on this page, but we should clarify this next work sesh.

Feel free to ask her or Vicky in #idol-design too!

> I'm not sure where this one is supposed to link to (I'm guessing the apply page), but the next button should probably link to https://webdev.cornelldti.org/. We can also confirm this with the designers.

It would link to a google form for that semester's Trends application.

Reply:
@JasonMun7 @cchrischen I'm asking Vannessa in Slack: https://cornelldti.slack.com/archives/C036VK3BRDE/p1728010585829349

Reply:
@JasonMun7 She replied. The buttons should turn dark on hover just like the other ones.